tests/DropboxTest.php

```diff
@@ -1,10 +1,92 @@
 <?php
-use PHPUnit\Framework\TestCase;
 
+namespace Kunnu\Dropbox\Tests;
+
+use GuzzleHttp\Psr7\Request;
+use Kunnu\Dropbox\Dropbox;
+use Kunnu\Dropbox\Models\FileMetadata;
+use Kunnu\Dropbox\Models\FolderMetadata;
+
+/**
+ * @author Cristiano Cinotti <cristianocinotti@gmail.com>
+ */
 class DropboxTest extends TestCase
 {
-    public function testDemo()
+    public function testGetMetadataOnFile()
+    {
+        $responseBody = <<<JSON
```

Comment:
We can delegate, the creation of dummy JSON responses for request, to a separate class(es). Thoughts?

Reply:
Great idea, test cases could be clearer. Maybe, we could have some traits, to do something like this:

```php
namespace Util 
{
    trait ResponsesForDropbox
    {
        public function getResponseForBlaBlaTest()
        {
             return new Response(......);
        }
    }
}

class DropboxTest extends TestCase
{
    use Util\ResponsesForDropbox;

    public function blaBlaTest()
    {
        $response = $this->getResponseForBlaBlaTest();
        ..............................
    }
}
```

Reply:
Neat. This makes sense 👌